attestation-agent/lib/src/token.rs

```diff
@@ -47,3 +55,22 @@ pub(crate) async fn get_kbs_host_from_cmdline() -> Result<String> {
 
     Ok(kbs_host)
 }
+
+pub(crate) async fn get_kbs_host_from_config_file() -> Result<String> {
+    // We only care about the aa_kbc_params value at the moment
+    #[derive(Debug, Deserialize)]
+    struct AgentConfig {
```

Comment:
I'm not sure whether a partial definition of the config will work. Try it my friend :P

Reply:
Yeah, I tried this is a dummy test environment and it seems to do fine in ignoring the rest of the toml file 😄